src/wayland.rs

```diff
@@ -77,12 +91,14 @@ impl Output {
         conn: &mut Connection<State>,
         global: &Global,
         gamma_manager: ZwlrGammaControlManagerV1,
+        name: Option<String>,
```

Comment:
This functions is always called with `name = None`, so let's remove this argument.